features/step_definitions/devops_steps.rb

```diff
@@ -1,10 +1,21 @@
-When(/^I run rake task "([^"]*)"$/) do |arg1|
-  # Write code here that turns the phrase above into concrete actions
-   require "rake"
-   @rake = Rake::Application.new
-   Rake.application = @rake
-   Rake::Task.define_task(:environment)
-   Rake.application.rake_require "tasks/scheduler"
-   @rake['fetch_github_commits'].invoke
+When(/^I run rake task "([^"]*)"$/) do |name|
+  # # Write code here that turns the phrase above into concrete actions
+  #  require "rake"
+  #  @rake ||= Rake::Application.new
+  #  Rake.application = @rake
+  #  Rake::Task.define_task(:environment)
+  #  Rake.application.rake_require "tasks/scheduler"
+  #  @rake[name].execute
+  $rake[name].execute
 end
 
+When(/^I run the rake task for calculating karma points$/) do
```

Comment:
Why aren't we re-using the previous step here? This one seems to be doing the same thing